### ISSUE TITLE
Paypal Pro API error

### ISSUE THREAD

**Issue by caseyzman** [akeeba/akeebasubs#174]
I've been using Subscriptions for a couple years now, and am now on v 4.2.1. I have recently been getting a "You do not have permissions to make this API call" when trying to use the "Paypal Payments Pro Direct" plugin. I've called Paypal and confirmed that the API and account are active and working, I went ahead and flushed the older API credentials and tried new ones with no success. Paypal is requesting that I send them NVP API logs although all I'm finding are the logs from "akpayment_paypalpaymentspro_ipn.php", which don't say much. DO you know where those API calls could be stored on a linux server?

**Comment by nikosdion** [comment]
Unfortunately we never had any way to actually *test* this blind implementation of the PayPal PPD API. The plugin was left in Akeeba Subscriptions based on user feedback that it works. I am actually not interested in maintaining it. If you cannot provide a code fix I would rather drop this plugin in a future version of Akeeba Subscriptions. The only plugins I use (and make sense for me to actively maintain) are 2Checkout, PayMill, VivaPayments and the classic PayPal API.

**Comment by caseyzman** [comment]
I was able to obtain the correlationID, sent it to paypal and they say the API credentials aren't being passed along, causing the "no permission" issue. I've tried it in sandbox (with sandbox credentials) as well as live, and on 2 different servers. I even tried replacing the current paypalpro plugin with the older version that previously worked, but that just broke something resulting in a blank payments page. I know that this won't be further supported, I just wanted to throw this information out there for anyone else. Thanks. Will post if I find anything further.
